### ISSUE TITLE
Collection creation

### ISSUE THREAD

**Issue by kdarras** [ecomontec/ecoSound-web#3]
add possibility to create collections in admin interface (similar to the add users function).

**Comment by kdarras** [comment]
Is it possible to remove this newly appearing dialog, @LiuDilongNJ ?


![Image](https://user-images.githubusercontent.com/7696198/195347771-7a4f39d4-d3a1-444b-886c-cf1db3c607f6.png)

**Comment by LiuDilongNJ** [comment]
Yes, I can do this. The previous programmer set the filter in the top left corner of the admin page (eg recordings, sites) to post request, and this dialog box will pop up when we refreshed the page. If you change it to get request, this dialog box will not pop up any more.

**Comment by kdarras** [comment]
working again
